internal/model/beer_media.go

```
@@ -82,9 +91,9 @@ func (s Slot) NextSlot() Slot {
 	}
 }
 
-func (s Slot) incrementSheetSlot(sheetSlot string) string {
+func (s Slot) incrementSheetSlot(sheetSlot string, rowSize int) string {
```

golangci-lint (2.5.0)

[error] unexported method "incrementSheetSlot" for struct "Slot" should be placed after the exported method "String"

(funcorder)